data-platform/autonomous-database/autonomous-json/python-duality-views/files/lab2_first_steps.py

```diff
@@ -0,0 +1,20 @@
+import oracledb
+
+#thick mode for SODA
+oracledb.init_oracle_client()
+
+#connection details
+connection = oracledb.connect(user="myapp",password="YOUR_PASSWORD",dsn="localhost:1521/FREEPDB1")
```

Comment:
Add a note that YOUR_PASSWORD should be changed to a real password!
Better change it to a command line option. Hardcoded values are bad.